tests/unit/service_test.py

```diff
@@ -99,14 +99,12 @@ def test_get_volumes_from_service_no_container(self):
 
     def test_split_domainname_none(self):
         service = Service('foo', image='foo', hostname='name', client=self.mock_client)
-        self.mock_client.containers.return_value = []
```

Comment:
These mock return values haven't been necessary for a while, since we refactored `_get_container_create_options()` to not make any api calls.